product-scenarios/4-application-development-using-apis/4.1-application-developers-should-be-able-to-use-apis-in-their-applications/4.1.3-delete-registered-application/src/test/java/org/wso2/am/scenario/tests/delete/registered/application/DeleteRegisteredApplicationNegativeTestCase.java

```diff
@@ -37,27 +38,55 @@ public void init() throws Exception {
 
     @Test(description = "4.1.3.5")
     public void testUnownedDeleteApplication() throws Exception {
-        HttpResponse addApplicationResponse = apiStore.addApplication(APPLICATION_NAME,
-                APIMIntegrationConstants.APPLICATION_TIER.UNLIMITED, "", "Description");
-        verifyResponse(addApplicationResponse);
-        assertEquals(new JSONObject(addApplicationResponse.getData()).get("status"), "APPROVED",
-                "Application creation failed for application: DeleteUnownedApplication");
-
+        addApplication(APPLICATION_NAME);
         apiStore.login(SUBSCRIBER2_USERNAME, SUBSCRIBER2_PW);
         HttpResponse deleteResponse = apiStore.removeApplication(APPLICATION_NAME);
         log.info("Delete unowned application Response Code : " + deleteResponse.getResponseCode());
         log.info("Delete unowned application Response Message : " + deleteResponse.getData());
         JSONObject responseData = new JSONObject(deleteResponse.getData());
         Assert.assertTrue(responseData.getBoolean("error"), "Error message received not received when" +
                 "deleting unowned application: " + deleteResponse.getData());
-        
+
+        apiStore.login(SUBSCRIBER_USERNAME, SUBSCRIBER_PW);
+        HttpResponse getApplicationsResponse = apiStore.getAllApplications();
+        log.info("Verify owner's application still exists in store response code : " +
+                getApplicationsResponse.getResponseCode());
+        log.info("Verify owner's application still exists in store response message : " +
+                getApplicationsResponse.getData());
+        assertTrue(getApplicationsResponse.getData().contains(APPLICATION_NAME),
+                "Application not available in store:" + APPLICATION_NAME);
+    }
+
+    @Test(description = "4.1.3.7", dependsOnMethods = {"testUnownedDeleteApplication"})
```

Comment:
It is better to provide a description for "description"

Reply:
The 'description' contains the description key, the description content corresponding to the key is available in the TESTCASES.md file.